sklearn/ensemble/weight_boosting.py

```diff
@@ -869,8 +869,8 @@ class AdaBoostRegressor(BaseWeightBoosting, RegressorMixin):
     ----------
     base_estimator : object, optional (default=None)
         The base estimator from which the boosted ensemble is built.
-        Support for sample weighting is required. If ``None``, then
-        the base estimator is ``DecisionTreeRegressor(max_depth=3)``
+        If ``None``, then the base estimator is
+        ``DecisionTreeRegressor(max_depth=3)``.
```

Comment:
```suggestion
        :class:`sklearn.tree.DecisionTreeRegressor(max_depth=3)`.
```

Reply:
will this work? let's see

Reply:
this doesn't work @glemaitre because there're some extra parameters

Reply:
Oh right. So let's merge.